k8ssandra/src/components/k8c-page.scss

```diff
@@ -0,0 +1,3 @@
+.k8c-page > .ItemListLayout > .header .title {
+  font-weight: normal;
```

Comment:
This, and the related cassdc-page.scss don't really do anything at the moment but serve as an example of how to take advantage of the styling integration that was added.